setup.py

```diff
@@ -350,73 +419,54 @@ def run(self):
         self._compile_webhost(self.webhost_dir)
 
 
-with open("README.md") as readme:
-    long_description = readme.read()
+class Clean(distutils.cmd.Command):
+    description = 'Clean up build generated files'
```

Comment:
Can be changed to a comment

Reply:
This description is use in the help documentation

```
➜ python setup.py --help-commands
/Users/varadmeru/work/microsoft/azfunctions/azfuncpyworker/setup.py:4: DeprecationWarning: The distutils package is deprecated and slated for removal in Python 3.12. Use setuptools or check PEP 632 for potential alternatives
  import distutils.cmd
Standard commands:
  build             build everything needed to install
  build_py          "build" pure Python modules (copy to build directory)
  build_ext         build C/C++ extensions (compile/link to build directory)
  build_clib        build C/C++ libraries used by Python extensions
  build_scripts     "build" scripts (copy and fixup #! line)
  clean             Clean up build generated files
  install           install everything from build directory
  install_lib       install all Python modules (extensions and pure Python)
  install_headers   install C/C++ header files
  install_scripts   install scripts (Python or otherwise)
  install_data      install data files
  sdist             create a source distribution (tarball, zip file, etc.)
  register          register the distribution with the Python package index
  bdist             create a built (binary) distribution
  bdist_dumb        create a "dumb" built distribution
  bdist_rpm         create an RPM distribution
  check             perform some checks on the package
  upload            upload binary package to PyPI
```